modules/ROOT/pages/setup.adoc

```diff
@@ -0,0 +1,196 @@
+= Anypoint Partner Manager Setup
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+This page describes the general requirements to set up and use Anypoint Partner Manager. You should be familiar with the following concepts:
+
+* EDI message formats.
+* EDI message schemas.
+* Transformations using Dataweave.
+* Handling X.509 Certificates (For AS2 transport protocol only).
+* Your own network of endpoints for listening to and sending messages.
+
+The ability to troubleshoot and diagnose errors produced in these areas and take or recommend corrective actions.
+
+=== Minimum Requirements
+
+Verify that you have the following:
+
+* Anypoint Platform user account with Anypoint Partner Manager access.
+* Appropriate roles allocated for users to access Anypoint Partner Manager. Roles are allocated via the 'Access Management' section in Anypoint Platform.
+
+_**Note to Maria** - Please add an image of the landing page of Anypoint Platform page that shows Partner Manager_
```

Comment:
Please hide this comment by adding // to the line start, or remove it completely. And also verify if that image needs to be added.

Reply:
Thanks for the tip on how to add comments into a file.